apps/channels/views.py

```diff
@@ -24,3 +25,21 @@ def new_whatsapp_message(request):
     message_data = json.dumps(request.POST.dict())
     tasks.handle_whatsapp_message.delay(message_data)
     return HttpResponse()
+
+
+@csrf_exempt
+def new_facebook_message(request: HttpRequest):
+    # https://developers.facebook.com/docs/messenger-platform/webhooks#:~:text=Validating%20Verification%20Requests
+    if request.method == "GET":
+        challenge = request.GET["hub.challenge"]
+        verify_token_exists = ExperimentChannel.objects.filter(
+            platform=ChannelPlatform.FACEBOOK, extra_data__verify_token=request.GET["hub.verify_token"]
+        ).exists()
```

Comment:
this seems like a good candidate for an manager method and a unit test:

```python
ExperimentChannel.objects.filter_by_extras(extra_key, extra_value, platform, team)
```

Reply:
https://github.com/dimagi/open-chat-studio/pull/115/commits/c668e30e2a35daf1c3bfe487cf0df07ed2a8d113